conferences/presentations/RIghtsCon-2021-06-07/script.md

```diff
@@ -0,0 +1,57 @@
+# Script for lightning talk at RightsCon2021
+
+8-10 minute recorded talk, in the Human Rights-Centered Design track - probably about 1500 words
+(possibly also with an AMA during the conference)
+
+## Original pitch
+([original application](https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/abstracts/2021-01-RightsCon.md))
+Data scientists make data-driven decisions throughout the lifecycle of their projects. 
+It is their duty to ensure that their work maintains a high level of moral integrity as well as the highest scientific standards, especially, when it impacts people’s lives. 
+In this presentation, we will introduce The Turing Way, an open source book project that involves and supports its diverse community in developing and sharing resources that make data science reproducible, ethical, collaborative and inclusive. 
+In particular, we will demonstrate the Turing Way guide for Ethical Research, the community behind its development, and its human rights-centred approach to data science ethics as a practice that includes activism for change. 
+Most importantly, we will demonstrate how RightsCon attendees can get involved!
+
+(based heavily on [this talk](https://zenodo.org/record/3968454#.YIA1Vmgo-i5) by Malvika Sharan, which was 10 minutes.
+
+## Skeleton:
+1. Intros and what is the Turing Way? (2 mins)
+3. What is the Turing Way Guide for Ethical Research? (5 mins)
+4. How can you get involved? (3 mins)
+
+## The Turing Way: Building a collaborative guide to ethical data science
+### Presenters: Malvika Sharan and Laura Carter
+
+(Introduce speakers)
+Data scientists make decisions throughout the lifecycle of their data-based projects.
+It is their duty to ensure that their work maintains a high level of moral integrity as well as the highest scientific standards, especially when it impacts people’s lives.
```

Comment:
```suggestion
As scientists, we have a duty to ensure that our work meets the highest scientific standards.
We also have an obligation to ensure that our work maintains the highest level of moral integrity: especially when it impacts people’s lives.
This impact on lives can be through the data we collect and use, or through the data products we build.
It can also be through the policies that we follow at our universities or institutions, and the working conditions that we create in our labs.

```